src/utils.ts

```diff
@@ -565,6 +575,7 @@ export async function safeFetch(url: string, options: RequestInit): Promise<Resp
     url: url,
     type: "basic",
     redirected: false,
+    bytes: () => Promise.resolve(new Uint8Array(0)),
```

Comment:
What's this for? Why do I get `The bytes property isn't part of the standard Response interface`?

Reply:
I got a type error here after upgrading TypeScript. Without change, the build will fail.

Did you npm install with the latest package.json in this PR and selected the latest TypeScript version in the IDE?

<img width="1272" alt="Screenshot 2024-12-27 at 11 03 01 PM" src="https://github.com/user-attachments/assets/ca18e224-765f-4d13-85ed-5553ca0983c3" />

Reply:
Oh yes I didn't manually set it in the IDE. Now it's fine 👍